static/bidder-params/taboola.json

```diff
@@ -36,5 +40,5 @@
       "type": "integer"
     }
   },
-  "required": ["tagid","publisherId"]
+  "required": ["publisherId"]
```

Comment:
To be clear, you no longer want to require a tag id?

Reply:
I would like to require any of both “tagId” or “tagid” (one at least)
 But no such option to configure that so currently its fine to set it as optional.

Reply:
You may represent it as:
```
"oneOf": [
  {
    "required": [ "tagid", "placementId" ]
  },
  {
    "required": [ "tagId", "placement_id" ]
  }
]
```

Reply:
@SyntaxNode  Thank you! did the change as you suggested :)